td-agent/msi/assets/td-agent.bat

```diff
@@ -3,4 +3,11 @@ set TD_AGENT_TOPDIR=%~dp0\..
 set PATH="%~dp0";%PATH%
 set FLUENT_CONF=%TD_AGENT_TOPDIR%\etc\td-agent\td-agent.conf
 set FLUENT_PLUGIN=%TD_AGENT_TOPDIR%\etc\td-agent\plugin
+set TD_AGENT_VERSION=%TD_AGENT_TOPDIR%\bin\td-agent-version.rb
+for %%p in (%*) do (
+    if "%%p"=="--version" (
+        ruby "%TD_AGENT_VERSION%"
+        exit 0

```

Comment:
We should remove this line because exit 0 causes unexpected Td-Agent Command Prompt shutdown.

Reply:
How about use `cmd /c "ruby %TD_AGENT_VERSION%"` here?

Reply:
Or, how about this?

```cmd
@echo off
set TD_AGENT_TOPDIR=%~dp0\..
set PATH="%~dp0";%PATH%
set FLUENT_CONF=%TD_AGENT_TOPDIR%\etc\td-agent\td-agent.conf
set FLUENT_PLUGIN=%TD_AGENT_TOPDIR%\etc\td-agent\plugin
set TD_AGENT_VERSION=%TD_AGENT_TOPDIR%\bin\td-agent-version.rb
for %%p in (%*) do (
    if "%%p"=="--version" (
        cmd /c "ruby %TD_AGENT_VERSION%"
        goto last
    )
)
"%TD_AGENT_TOPDIR%\bin\fluentd" %*

:last
```

Reply:
It looks good!